src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -33,9 +33,10 @@
          that check the invoked process name against known casing which fail under code coverage.
          This is probably because the code coverage target invocation does not canonicalize the
          process name. -->
-    <TargetOS Condition="'$(TargetOS)'==''">$(OS)</TargetOS>
-    <TestHostExecutable Condition="'$(TargetOS)'!='Windows_NT'">./corerun</TestHostExecutable>
-    <TestHostExecutable Condition="'$(TargetOS)'=='Windows_NT'">CoreRun.exe</TestHostExecutable>
+    <!-- Additionally, note that often at this point neither TargetOS nor OSEnvironment is  specified, so we must choose using OS for now.
```

Comment:
I will investigate some more.  I thought the find/replace was a bit hacky for my tastes but I can definitely explore that too.